ghcide.cabal

```diff
@@ -276,7 +276,7 @@ executable ghcide
         hashable,
         haskell-lsp,
         haskell-lsp-types,
-        hie-bios >= 0.6.0 && < 0.8,
+        hie-bios >= 0.7.1 && < 0.8,
```

Comment:
We should probably remove those as they are redundant

Reply:
Yeah, they are already in the lib